tic-tac-toe.js

```diff
@@ -1,7 +1,221 @@
 function TicTacToe() {
+  this.board = [["","",""], ["","",""], ["","",""]];
+  this.count = 0;
+  }
 
-}
+  var checkSpot = function(col, row, sign , board) {
+    if (board[col][row] === sign) {
+      return true;
+    } else {return false;}
+  }
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.CheckRow = function(col, row, sign) {
+    if (row === 0) {
+      if (checkSpot(col, 1, sign, this.board) && (checkSpot(col,2, sign, this.board))) {
+        return true;
+    } else if ( row === 1) {
+      if (checkSpot(col, 0, sign, this.board) && (checkSpot(col,2, sign, this.board)))
+        return true;
+    } else {
+      if (checkSpot(col, 0, sign, this.board) && (checkSpot(col,1, sign, this.board)))
```

Comment:
It seems like the logic for these three conditionals could more easily be done one if using the `row` variable. I'd like to see you refactor this conditional to execute the `checkSpot` method only once.